### ISSUE TITLE
NTR: 2'-deoxyadenosine (dAdo) elevated in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8822]
New term request 2'-deoxyadenosine (dAdo) elevated in urine (segal_080312143155):   UMLS: 1863255

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000123
- label: Elevated urinary 2'-deoxyadenosine level
- parents: [Abnormal urinary purine level (HP:0033161)](https://hpo.jax.org/app/browse/term/HP:0033161)
- pmid: 
- def: The amount of 2'-deoxyadenosine (dAdo) in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8822
- synonyms: Elevated urinary dAdo level(NONE)|2'-deoxyadenosine (dAdo) elevated in urine(NONE)
